openlibrary/coverstore/code.py

```diff
@@ -132,7 +132,7 @@ def error(code__msg):
             raise e
 
         source_url = i.source_url
-        data = i.data
+        data = web.data()
```

Comment:
Why this change?

Reply:
Because we are now sending the data in requests body not in url parameters - this is the way to fetch the data from request body in `web.py` framework.

Reply:
Nice!